### PR TITLE
fix: notification library

### PR DESCRIPTION
**Pull request by nicolasburtey**

### CODE REVIEW

File: app/navigation/local-notification-wrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { useApolloClient } from "@apollo/client"
+import { useIsAuthed } from "@app/graphql/is-authed-context"
+import { addDeviceToken, hasNotificationPermission } from "@app/utils/notifications"
+import messaging, { FirebaseMessagingTypes } from "@react-native-firebase/messaging"
+import { useEffect } from "react"
+
+export const NotificationWrapper = (): JSX.Element => {
```
Comment (UncleSamtoshi) [inline app/navigation/local-notification-wrapper.tsx:7]: This doesn't appear to be a wrapper, but just a component that handles the notification logic.

File: app/navigation/app-state.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    },
+    [client, isAuthed],
+  )
+
+  useEffect(() => {
+    const subscription = AppState.addEventListener("change", handleAppStateChange)
+    return () => subscription.remove()
+  }, [handleAppStateChange])
+
+  return <>{children}</>
```
Comment (UncleSamtoshi) [inline app/navigation/app-state.tsx:36]: It it necessary for AppStateWrapper to pass through `<>{children}</>` and be a parent component of them, or could it just be a sibling component in `app.tsx`.

**Review by UncleSamtoshi — DISMISSED** [review]

File: app/navigation/local-notification-wrapper.tsx (hunk 1 above)
Comment (nicolasburtey) [reply app/navigation/local-notification-wrapper.tsx:7]: any suggestion of name we should give to such component?

File: app/navigation/app-state.tsx (hunk 2 above)
Comment (nicolasburtey) [reply app/navigation/app-state.tsx:36]: yeah good can be just a sibling

**Review by Extheoisah — APPROVED** [review]

**Pull request merged**